src/Raven.Studio/languageService/test/grammar/loadStatement.spec.ts

```diff
@@ -0,0 +1,54 @@
+﻿import {parseRql} from "../../src/parser";
+import {
+    LoadDocumentByNameContext,
+    LoadStatementContext
+} from "../../src/generated/RqlParser";
+
+describe("LOAD statement parser", function () {
+    it("single", function () {
+        const {parseTree, parser} = parseRql("from test load x as y");
+
+        expect(parser.numberOfSyntaxErrors)
+            .toEqual(0);
+
+        const load = parseTree.loadStatement();
+        expect(load)
+            .toBeInstanceOf(LoadStatementContext);
+        expect(load._item._name.text)
+            .toEqual("x");
+        
+        expect(load._item._as.text).toEqual("asy");
+    });
+
+    it("array", function () {
+        const {parseTree, parser} = parseRql("from test load t as t, t as t");
```

Comment:
can we avoid duplicates here? a as mappedA, b as mappedB

Reply:
Duplicates a lot of code and changed nothing, we assert subtrees.